File: Trials and Triangulations/trials.c
#include <stdio.h>
#include <math.h>

int main() {
    int a, b, c;
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);

    double s = (a + b + c) / 2.0;
    double A = sqrt(s * (s - a) * (s - b) * (s - c));
    printf("%.14f", A);

    return 0;
}

File: Trials and Triangulations/trials.java
import java.util.Scanner;

public class trials {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = scanner.nextInt();

        double s = (a + b + c) / 2.0;
        double A = Math.sqrt(s * (s - a) * (s - b) * (s - c));
        System.out.printf("%.14f", A);
    }
}
